dataclients/kubernetes/testdata/ingressV1/service-ports-endpointslices/foo.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  labels:
+    app: myapp
+  name: myapp-ingress
+  namespace: myapp-ns
+spec:
+  rules:
+    - host: example.org
+      http:
+        paths:
+          - backend:
+              service:
+                name: myapp-service
+                port:
+                  number: 80
+            pathType: ImplementationSpecific
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: myapp
+  name: myapp-service
+  namespace: myapp-ns
+spec:
+  clusterIP: 10.5.69.228
+  clusterIPs:
+    - 10.5.69.228
+  internalTrafficPolicy: Cluster
+  ipFamilies:
+    - IPv4
+  ipFamilyPolicy: SingleStack
+  ports:
+    - port: 80
+      protocol: TCP
+      targetPort: http-grafana
```

Comment:
Here service uses named target port which could only be resolved by reading pod definition.

Endpoints logic has a special condition for a single port:
https://github.com/zalando/skipper/blob/52ef4ef61928da148983bf9aca40cc893d99baa6/dataclients/kubernetes/ingressdefinitions.go#L91-L99